pkg/services/ngalert/notifier/alertmanager.go

```diff
@@ -26,13 +26,12 @@ import (
 )
 
 const (
-	// maintenanceNotificationAndSilences how often should we flush and garbage collect notifications
-	notificationLogMaintenanceInterval = 15 * time.Minute
-)
+	// How often we flush and garbage collect notifications and silences.
+	maintenanceInterval = 15 * time.Minute
 
-// How long should we keep silences and notification entries on-disk after they've served their purpose.
-var retentionNotificationsAndSilences = 5 * 24 * time.Hour
-var silenceMaintenanceInterval = 15 * time.Minute
+	// How long we keep silences on disk after they've expired.
```

Comment:
We stopped persisting to disk in https://github.com/grafana/grafana/pull/84706, maybe we should fix up this comment while we're here?